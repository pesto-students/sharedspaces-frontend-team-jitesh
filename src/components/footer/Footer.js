import React from 'react'
import './footer.scss'

export default function Footer() {
    return (
        <div className='footer text-center'>
            <div className='border-t-2 border-gray-700 flex justify-between py-10'>
                <div className="left">
                    <img className='logo' src="/assets/images/logo.png" alt="" />
                </div>
                <div className="right">
                    <nav className="flex sm:justify-center space-x-4 ml-10">
                        {[
                            ['Home', '/'],
                            ['Explore', '/search'],
                            ['About', '/'],
                            ['Contact', '/'],
                        ].map(([title, url]) => (
                            <a href={url} className="rounded-lg px-3 py-2 text-slate-700 font-medium hover:text-slate-100 transition duration-150 ease-in-out ">{title}</a>
                        ))}
                    </nav>
                </div>
            </div>
            <p className="text-gray-700">
                SharedSpace | copyright &#169; 2022
            </p>
        </div>
    )
}
