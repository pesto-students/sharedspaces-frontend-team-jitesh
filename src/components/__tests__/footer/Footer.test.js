import { render, screen, cleanup } from "@testing-library/react";
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer'
import Footer from '../../footer/Footer'

afterEach(() => {
    cleanup()
})


test('test matches snapshot', () => {
    const component = renderer.create(<BrowserRouter><Footer /></BrowserRouter>)

    let tree = component.toJSON()
    expect(tree).toMatchSnapshot();
})
