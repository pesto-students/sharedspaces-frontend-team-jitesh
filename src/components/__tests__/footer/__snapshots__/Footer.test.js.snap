// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test matches snapshot 1`] = `
<div
  className="footer text-center"
  data-testid="footer"
>
  <div
    className="footer-with-menu border-t-2 border-gray-700 py-10"
  >
    <div
      className="left"
    >
      <img
        alt=""
        className="logo"
        src="/assets/images/logo.png"
      />
    </div>
    <div
      className="right"
    >
      <nav
        className="right-menu"
      >
        <a
          className="rounded-lg pl-10 py-2 text-slate-700 font-medium hover:text-slate-100 transition duration-150 ease-in-out "
          href="/"
          onClick={[Function]}
        >
          Home
        </a>
        <a
          className="rounded-lg pl-10 py-2 text-slate-700 font-medium hover:text-slate-100 transition duration-150 ease-in-out "
          href="/search"
          onClick={[Function]}
        >
          Explore
        </a>
        <a
          className="rounded-lg pl-10 py-2 text-slate-700 font-medium hover:text-slate-100 transition duration-150 ease-in-out "
          href="/"
          onClick={[Function]}
        >
          About
        </a>
        <a
          className="rounded-lg pl-10 py-2 text-slate-700 font-medium hover:text-slate-100 transition duration-150 ease-in-out "
          href="/"
          onClick={[Function]}
        >
          Contact
        </a>
      </nav>
    </div>
  </div>
  <p
    className="text-gray-700"
  >
    SharedSpace | copyright Â© 2022
  </p>
</div>
`;
